libraries/lambda_handlers/client_converter_callback_handler.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import unicode_literals, print_function
+from libraries.lambda_handlers.handler import Handler
+from libraries.client.client_converter_callback import ClientConverterCallback
+
+
+class ClientConverterCallbackHandler(Handler):
+
+    def _handle(self, event, context):
+        """
+        :param dict event:
+        :param context:
+        :return dict:
+        """
+        # Execute
```

Comment:
Use the retrieve data function here to get the params so if not passed the caller doesn't get some weird coding error

Reply:
This is the format of the returned data from the converter:

            callback_results = {
                'job_id': self.job_id,
                'results': results
            }